controllers/chat/ChatController.js

```diff
@@ -159,7 +159,7 @@ module.exports = {
     const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
     try {
       await client.connectUser({id: req.userId}, req.token);
-      let channels = await client.queryChannels({type: 'messaging', members: {$in: [req.userId]}}, [
```

Comment:
ini type messagingnya knp di remove ya mas ? apakah by default sudah type messaging ?

Reply:
mas @brm-stnd this is for fetching all type of channel (i.e messaging, type, and group). Adding type: 'messaging' in the filter will only returns messaging, while we need to fetch all of the three types. Thus, the messaging filter is removed